http-gateway/web/src/common/services/fetch-api.js

```diff
@@ -17,12 +17,8 @@ export const errorCodes = {
 
 export const fetchApi = async (url, options = {}) => {
   const { audience, scopes, body, timeToLive, ...fetchOptions } = options
-  const { audience: defaultAudience } = security.getWebOAuthConfig()
   // Access token must be gathered and set as a Bearer header in all requests
-  const accessToken = await security.getAccessTokenSilently()({
-    audience: audience || defaultAudience,
```

Comment:
Support for the audience shouldn't be dropped. In case it's present in the `./well-known/configuration` endpoint, it should be part of every token request (configuration of oidc module) as well as part of the request to get the authorization code ( + New Device ).